src/server.js

```diff
@@ -29,7 +31,7 @@ app.use(expressValidator({
 }));
 app.use(bodyParser.json());
 
-if (config.get('env') !== 'test') {
+if (config.get('env') !== 'test' || process.env.FORCE_OUTPUT) {
```

Comment:
to be able to see the output in tests
The alternative is to have an env like "QUIET=1" to remove the logger, and set it in travis.

That said, I usually prefer to have more log than less log :)